src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlStreamNodeWriter.cs

```diff
@@ -267,8 +267,7 @@ protected void WriteUTF8Chars(byte[] chars, int charOffset, int charCount)
         {
             if (charCount < bufferLength)
             {
-                int offset;
-                byte[] buffer = GetBuffer(charCount, out offset);
+                byte[] buffer = GetBuffer(charCount, out int offset);
```

Comment:
Do we still need this WriteUTF8Chars method now that there's the WriteUTF8Bytes method below?  Any remaining call sites could be changed from `WriteUTF8Chars(value, offset, count)` to `WriteUTF8Chars(value.AsSpan(offset, count))`.